README.md

```diff
@@ -3,60 +3,79 @@
+a foundation for our projects at [JoliCode](https://jolicode.com/).
 
 Learn more about it and why we created and open-sourced this starter-kit in our blog posts:
 [in English 🇬🇧](https://jolicode.com/blog/introducing-our-docker-starter-kit) or
 [in French 🇫🇷](https://jolicode.com/blog/presentation-de-notre-starter-kit-docker).
 
 > **Warning**
-> You are reading the README of version 3 that uses [Invoke](https://www.pyinvoke.org/).
+> You are reading the README of version 4 that uses [castor](https://github.com/jolicode/castor).
 
+* If you are using [Invoke](https://www.pyinvoke.org/), you can read the [dedicated README](https://github.com/jolicode/docker-starter/tree/v3.0.0);
+* If you want to migrate from docker-starter v2.x to v3.0, you can read the [dedicated guide](./UPGRADE-4.0.md);
 * If you are using [Fabric](https://www.fabfile.org/), you can read the [dedicated README](https://github.com/jolicode/docker-starter/tree/v2.0.0);
-* If you want to migrate from docker-starter v2.x to v3.0, you can read the [dedicated guide](./UPGRADE-3.0.md);
 
 ## Project configuration
 
 Before executing any command, you need to configure a few parameters in the
-`invoke.py` file:
+`.castor.php` file, in the `create_default_context()` function:
 
-* `project_name` (**required**): This will be used to prefix all docker
-objects (network, images, containers);
+* `project_name` (**required**): This will be used to prefix all docker objects
+(network, images, containers);
 
-* `root_domain` (optional, default: `project_name + '.test'`): This is the
-root domain where the application will be available;
+* `root_domain` (optional, default: `project_name + '.test'`): This is the root
+domain where the application will be available;
 
-* `extra_domains` (optional): This contains extra domains where the
-application will be available;
+* `extra_domains` (optional): This contains extra domains where the application
+will be available;
 
 * `project_directory` (optional, default: `application`): This is the host
-directory containing your PHP application.
+directory containing your PHP application;
+
+* `php_version` (optional, default: `8.2`): This is PHP version.
 
 For example:
 
-```py
-project_name = 'foobar'
-root_domain = project_name + '.test'
-extra_domains = ['api.' + root_domain]
-project_directory = 'application'
+```php
+#[AsContext(default: true)]
+function create_default_context(): Context
+{
+    $projectName = 'my-project';
+    $tld = 'dev';
+
+    $c = [
+        'project_name' => $projectName,
+        'root_domain' => "$projectName.$tld",
+        'extra_domains' => [
+            "www.$projectName.$tld",
+            "admin.$projectName.$tld",
+            "api.$projectName.$tld",
+        ],
+        'php_version' => 8.2,
+
 ```
 
-Will give you `https://foobar.test` and `https://api.foobar.test` pointing at your `application/` directory.
+Will give you `https://my-project.dev`,  `https://www.my-project.dev`,
```

Comment:
Why do we move to .dev TLD? As it's a real TLD, I'm not confortable to use it by default.

Reply:
It's just a sample in the README. Do you want me to change?

Reply:
Fixed in https://github.com/jolicode/docker-starter/pull/219/commits/acc1f7b059da59a2488b6fd6996d05a649f226b6